### PR TITLE
CIS tests for if service accounts are local admins

### PR DESCRIPTION
**Pull request by tboggiano**
# A New PR

THANK YOU - We love to get PR's and really appreciate your time and help to improve this module

## Accepting a PR

Before we accept the PR - please confirm that you have run the tests locally to avoid our automated build and release process failing. You can see how to do that in our wiki

https://github.com/sqlcollaborative/dbachecks/wiki 

## Please confirm you have 0 failing Pester Tests

[X] There are 0 failing Pester tests

## Changes this PR brings

- CIS test for SQL Server Engine, SQL Agent, and Full Text start up accounts to not be in the local administrators group

Please add below the changes that this PR covers. It doesnt need an essay just the bullet points :-)

### CODE REVIEW

File: internal/configurations/configuration.ps1 [hunk 1]
```diff
@@ -241,6 +241,9 @@ Set-PSFConfig -Module dbachecks -Name skip.instance.latestbuild -Validation bool
 Set-PSFConfig -Module dbachecks -Name skip.security.sadisabled -Validation bool -Value $true -Initialize -Description "Skip the check for if the sa login is disabled"
 Set-PSFConfig -Module dbachecks -Name skip.security.saexist -Validation bool -Value $true -Initialize -Description "Skip teh check for a login named sa does not exist"
 Set-PSFConfig -Module dbachecks -Name skip.security.containedbautoclose -Validation bool -Value $true -Initialize -Description "Skips the scan for contained databases should have auto close enabled"
+Set-PSFConfig -Module dbachecks -Name skip.security.engineserviceadmin -Validation bool -Value $true -Initialize -Description "Skips the scan for contained databases should have auto close enabled"
+Set-PSFConfig -Module dbachecks -Name skip.security.agentserviceadmin -Validation bool -Value $true -Initialize -Description "Skips the scan for contained databases should have auto close enabled"
+Set-PSFConfig -Module dbachecks -Name skip.security.fulltextserviceadmin -Validation bool -Value $true -Initialize -Description "Skips the scan for contained databases should have auto close enabled"
```
Comment (SQLDBAWithABeard) [inline internal/configurations/configuration.ps1]: The descriptions :-)

**Comment by tboggiano** [comment]
This also takes care of PR #701

File: internal/configurations/configuration.ps1 (hunk 1 above)
Comment (tboggiano) [reply internal/configurations/configuration.ps1]: How terrible of a copy paste job, so embarrassing.

Comment (tboggiano) [reply internal/configurations/configuration.ps1]: all better, I hope.

**Comment by tboggiano** [comment]
Close 662
Close 663
Close 664

**Comment by SQLDBAWithABeard** [comment]
done under #749

**Pull request closed without merging**